R/user-access.R

```diff
@@ -0,0 +1,39 @@
+#' Get information about when users last accessed a dataset
+#'
+#' Useful to get dataset-user interaction metadata about all of the users that
+#' have access to a particular dataset.
+#'
+#' @param dataset_entity a dataset object or a dataset entity (e.g. a single
+#' entry from `selectDatasetCatalog()`)
+#' @param user_fields metadata fields about users to include in the dataframe
+#' (default: c("name", "email", "access_time"))
+#'
+#' @return a dataframe with information about all users for a particular dataset
+#' @export
+userInfo <- function (dataset_entity, user_fields = c("name", "email", "access_time")) {
```

Comment:
`dataset` not `dataset_entity` for consistency.

Reply:
Also, the function that gets the `UserCatalog` from a dataset should be called `users()`, for consistency with e.g. `datasets()`, `variables()`, `projects()`, etc. The conventions help us compose new features, and in principle make it easier for our users to figure out what a function would be called.

If you want to add a special print method for it, or a function (like this) that wraps and prints a special summary, that's fine.